jobsdb/jobsdb.go

```diff
@@ -1467,6 +1469,7 @@ func (jd *Handle) createDSTablesInTx(ctx context.Context, tx *Tx, newDS dataSetT
 		custom_val VARCHAR(64) NOT NULL,
 		event_payload `+string(columnType)+` NOT NULL,
 		event_count INTEGER NOT NULL DEFAULT 1,
+		payload_size INTEGER NOT NULL DEFAULT 0,
```

Comment:
What about existing tables that don't have the column?

Reply:
I forgot to add the migration template!